web/src/pages/Cases/CaseDetails/Overview/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "styled-components";
+import { useParams } from "react-router-dom";
+import { formatEther } from "viem";
+import { useDisputeDetailsQuery } from "queries/useDisputeDetailsQuery";
+import { useDisputeTemplate } from "queries/useDisputeTemplate";
+import { useCourtPolicy } from "queries/useCourtPolicy";
+import { Periods } from "consts/periods";
+
+import DisputeInfo from "components/DisputeCard/DisputeInfo";
+import Verdict from "components/Verdict/index";
+import { useVotingHistory } from "hooks/queries/useVotingHistory";
+import { getLocalRounds } from "utils/getLocalRounds";
+import { DisputeBasicInfo } from "./DisputeBasicInfo";
+import { Policies } from "./Policies";
+
+const Container = styled.div`
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  gap: calc(16px + (32 - 16) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+  padding: calc(16px + (32 - 16) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+
+  > h1 {
+    margin: 0;
+  }
```

Comment:
which component is this style for?

Reply:
This was for the title, moved it to DisputeContext now